packages/neuron-wallet/src/main.ts

```diff
@@ -88,6 +88,9 @@ function createWindow() {
   monitorChain(mainWindow.webContents)
   const terminalChannel = new TerminalChannel(mainWindow.webContents)
   terminalChannel.start()
+  BrowserWindow.addDevToolsExtension(
+    `/Users/ChenYu/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.6.0_0`,
```

Comment:
Never use hardcoded local path that's specific to a computer.

Reply:
unbelievable mistake, and lucky that no more info leaks. Will never do it again.